### ISSUE TITLE
"Skip to main content" visible on all pages

### ISSUE THREAD

**Issue by gjost** [denshoproject/encyc-tng#75]
The "Skip to main content" accessibility link should be hidden unless the user is using a screen reader but appears visually on all pages.

**Comment by nicklee** [comment]
This is skip-link.scss, it's included in main.scss, but looks like staging hasn't compiled the stylesheets correctly so the styles haven't been included. 

I can confirm this works locally, so suspect like a lot of the other issues this is a compiling issue on staging.

**Comment by gjost** [comment]
Yeah I bet if we can solve this it will take care of a bunch of issues

**Comment by nicklee** [comment]
Looks like this has been resolved with your latest updates 🥳

**Comment by gjost** [comment]
Should've closed this a long time ago.
